sdk/monitor/monitor-ingestion/test/public/logsIngestionClient.spec.ts

```diff
@@ -56,7 +57,7 @@ describe("LogsIngestionClient live tests", function () {
   });
 
   it("sends basic data", async function () {
-    if (isPlaybackMode()) this.skip();
+    if (isPlaybackMode() && isNode) this.skip();
```

Comment:
is this because of the OS-specific behavior for gzip? Maybe we can centralize this check?

Reply:
Yes moved this check to the describe() portion of test file.
Yes it is becuase the gzipping is OS-specific